plugins/Number.py

```diff
@@ -46,11 +46,27 @@ def init(self, logger):
 '''Check that the value is accurate. Consider whether another tag should
 be used if the value is valid.''')
         )
+        self.errors[3093] = self.def_class(item = 3091, level = 2, tags = ['value', 'fix:chair'],
+            title = T_('Suspicious value'),
+            detail = T_(
+'''The tag expects a positive, round number without unit'''),
+            fix = T_(
+'''Check that the value is accurate. Consider whether another tag should
+be used if the value is valid.''')
+        )
+
+        self.tag_number = ["height", "width", "length", "ele"]
+        self.tag_number_integer = ["population", "admin_level"] # Only positive integers (no units) allowed
+        tag_number_directional = ["maxheight", "maxheight:physical", "maxwidth", "maxlength", "maxweight", "maxspeed", "minspeed"]
+
+        # Add suffixes to the directional tags, add everything to tag_number
+        for i in ["", ":forward", ":backward"]:
```

Comment:
The `""` does not make duplicate in the list ?

Reply:
It shouldn't, because I'm copying from `tag_number_directional` (a local variable just to have the items that need suffixes) to `self.tag_number`. The eventual content of `self.tag_number` is `['diameter', 'distance', 'ele', 'height', 'length', 'width', 'maxaxleload', 'maxheight', 'maxheight:physical', 'maxlength', 'maxspeed', 'maxspeed:advisory', 'maxweight', 'maxwidth', 'minspeed', 'maxaxleload:forward', 'maxheight:forward', 'maxheight:physical:forward', 'maxlength:forward', 'maxspeed:forward', 'maxspeed:advisory:forward', 'maxweight:forward', 'maxwidth:forward', 'minspeed:forward', 'maxaxleload:backward', 'maxheight:backward', 'maxheight:physical:backward', 'maxlength:backward', 'maxspeed:backward', 'maxspeed:advisory:backward', 'maxweight:backward', 'maxwidth:backward', 'minspeed:backward', 'admin_level', 'capital', 'heritage', 'population', 'step_count']` 

p.s.: I did it this way so someone could easily add i.e. `:hgv`, like
`for i in ["", ":forward", ":backward", ":hgv"]:`
to also test for i.e. `maxspeed:hgv` (and identical for other common items; I'm considering `:hgv`, `:bus` and `:trailer` as they're very common to be special-cases)

Reply:
My bad, it is ok.